CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+## Flaky and Skip Changes to lookup.json
+The process for adding flaky or skip tags to the lookup.json is reasonably simple. Make your changes put together a PR with ideally a link to the community CI failing and then it will be reviewed.
+
+
+## Submitting a module to CitGM
```

Comment:
I don't think we'd want to duplicate this, a link to the `README` would work (and I think the readme is the best place for this stuff, it's not just for contributors).